incubator/build/src/platforms/windows.ts

```diff
@@ -0,0 +1,127 @@
+import * as fs from "node:fs/promises";
+import * as os from "node:os";
+import * as path from "node:path";
+import type { Ora } from "ora";
+import { retry } from "../async";
+import { makeCommand } from "../command";
+
+type PackageInfo = {
+  packageName: string;
+  appId: string;
+};
+
+function makePowerShell(
+  cwd: string,
+  elevated = false
+): ReturnType<typeof makeCommand> {
+  const withPowerShell = { cwd, shell: "powershell" };
+  const prelude = [
+    "-NoLogo",
+    "-NoProfile",
+    "-NonInteractive",
+    "-ExecutionPolicy",
+    "Bypass",
+    "-Command",
+  ];
+  if (elevated) {
+    const start = makeCommand("Start-Process", withPowerShell);
+    const startArgs = prelude.join(" ");
+    return (...args: string[]) =>
+      start(
+        "-FilePath",
+        "powershell",
+        "-ArgumentList",
+        `'${startArgs} "${args.join(" ")}"'`,
+        "-PassThru",
+        "-Verb",
+        "RunAs"
+      );
+  } else {
+    const ps = makeCommand("powershell", withPowerShell);
+    return (...args: string[]) => ps(...prelude, `"${args.join(" ")}"`);
+  }
+}
+
+async function getPackageInfo(app: string): Promise<PackageInfo | null> {
+  const filename = path.join(app, "AppxManifest.xml");
+  const manifest = await fs.readFile(filename, { encoding: "utf-8" });
+  const m = manifest.match(/<Identity[^>]*Name="(.*?)"/s);
```

Comment:
```suggestion
  const m = manifest.match(/<Identity[^>]+Name="(.*?)"/s);
```
At least one char between Identity and Name? Also, maybe an XML parser instead? I know... but its safer and less error prone.

Reply:
And there are other instances below where it would be handy. readAppxManifest -> json object (from xml reader)

Reply:
Yeah… I guess we'll have to do this. I'll fix it in a separate PR today.